lib/swedbankpay_checkout/client/psp/payment_orders.ex

```diff
@@ -45,6 +45,25 @@ defmodule SwedbankpayCheckout.Client.Psp.PaymentOrders do
     })
   end
 
+  @doc """
+  Get a payment order with expanded payer info, the payment_order_id should be the full id, with paths, not truncated to the UUID
+  """
+  @spec get_payment_order_with_payer(Tesla.Env.client(), String.t()) ::
+          {:ok, PaymentOrders.OrderResponse.t()}
+          | {:error, Tesla.Env.t()}
+          | {:error, {String.t(), Tesla.Env.t()}}
+  def get_payment_order_with_payer(client, payment_order_id) do
```

Comment:
How hard would it be to send in a (possibly empty) array of extra fields to expand to `get_payment_order`?
Problems with response decoding?

That'd be the smoothest API to use for us, but perhaps not worth effort in the short term.